packages/block-library/src/media-text/edit.native.js

```diff
@@ -259,23 +311,60 @@ class MediaTextEdit extends Component {
 			setAttributes( { verticalAlignment: alignment } );
 		};
 
+		const controls = (
+			<InspectorControls>
+				<PanelBody title={ __( 'Media & Text settings' ) }>
+					<ToggleControl
+						label={ __( 'Crop image to fill entire column' ) }
+						checked={ imageFill }
+						onChange={ () =>
+							setAttributes( {
+								imageFill: ! imageFill,
+							} )
+						}
+					/>
+				</PanelBody>
+			</InspectorControls>
+		);
+
 		return (
 			<>
+				{ mediaType === MEDIA_TYPE_IMAGE && controls }
```

Comment:
WDYT about not creating the controls before the `return` and create them only when we need it? Right now we always create them but we use them only when `mediaType` is equal the `MEDIA_TYPE_IMAGE`

Reply:
Good idea! I moved `controls` out of the render method